homework/hackyourtemperature/__test__/app.test.js

```diff
@@ -0,0 +1,42 @@
+import app from "../app.js";
+import supertest from "supertest";
+
+const request = supertest(app);
+
+describe("POST /", () => {
+  describe("user enters a cityName", () => {
+    it("should response 200 when containing a cityName", async() => {
+      const response = await request.post('/weather').send({
+        cityName: "Cairo"
+      })
+      expect(response.statusCode).toBe(200)
+    });
+
+    it("should specify json in the content", async() => {
```

Comment:
Messages should be declarative and clear about their content. Imagine someone is looking for more understanding of the code, this is one of the places he will look at to know how the code work, and it's for developers, so technicality is not an issue.

```suggestion
    it("Content-Type should satisfy a `json` type", async () => {
```

And don't forget to format your document as appropriet, look for your device's shortcut
https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode

Reply:
done